frontend/amp/components/elements/RichLinkBlockComponent.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { css } from 'emotion';
+import { palette } from '@guardian/pasteup/palette';
+import { serif, sans } from '@guardian/pasteup/fonts';
+import { pillarPalette } from '@frontend/lib/pillars';
+
+export const RichLinkBlockComponent: React.SFC<{
+    element: RichLinkBlockElement;
+    pillar: Pillar;
+}> = ({ element, pillar }) => {
+    if (element.sponsorship) {
+        throw new Error('Sponsored rich links not supported');
```

Comment:
Is this something we should (eventually) support in AMP or not? Just trying to get a handle on what the appropriate error handling should be here.

Reply:
I'm not sure if we use this or where, but we should handle somewhat differntly than this.